vm/v8vm/v8/libjs/environment.js

```diff
@@ -1,6 +1,13 @@
 'use strict';
 
 (function () {
+    function isRegExp(regStr) {
+        if (typeof regStr === 'object') {
```

Comment:
For checking the variable is regular expression or not, `typeof regStr === 'object'` could be misleading. 
How about this? `if (regStr instanceof RegExp)`